source/Octodiff/Octodiff.csproj

```diff
@@ -29,7 +29,7 @@
     <Reference Include="System" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.5' ">
     <PackageReference Include="Microsoft.NETCore.Runtime.CoreCLR" Version="2.0.0" />
```

Comment:
Missed this the first time, these first two packages should be removed as they are exe runtime packages.

It looks like it is now adding a reference to `NetStandard.Library` directly (in the nuspec). Most of the packages below are included in that, but the original advice was that you should only include the minimum subset of packages to run your lib.

I'm on the fence whether to add `DisableImplicitFrameworkReferences` (https://gist.github.com/thecodejunkie/f87c2b680f9235f33a0543b53ece1d6e) to revert this behaviour of netstandard 1.x, or remove the unneeded package references.

In 2.0 the advice is to always allow the implicit includsion of `NETStandard.Library`.